packages/client/src/__tests__/tsa/index.test.ts

```diff
@@ -63,20 +64,9 @@ describe('TSAClient', () => {
       });
 
       it('returns an error', async () => {
-        await expect(subject.createTimestamp(signature)).rejects.toThrow(
-          InternalError
-        );
-      });
-
-      it('has the expected error code', async () => {
-        await subject
-          .createTimestamp(signature)
-          .then(() => {
-            throw Error('expected error');
-          })
-          .catch((err) => {
-            expect(err.code).toEqual('TSA_CREATE_TIMESTAMP_ERROR');
-          });
```

Comment:
The helper keeps us from writing tests like this where we have to write a dedicate test case to catch the error and check the code.